src/Dynamic/Includes/cards/taskcard.pug

```diff
@@ -0,0 +1,10 @@
+.card.is-box-shadow
+    .card-content
+        .media
+            .media-content
+                p.title.is-3 #{task.name}
+                p.subtitle.is-6
+                    | Next Run: #{task.nextInvocation}
```

Comment:
Could we also show when it was last run and if it was successful, or is this data not kept?

Reply:
Unfortunately the lib doesn't keep the last run and state.
We might extend the Job structure with logging and db store for this feature.

Reply:
Is that something you'd be comfortable doing? -- if yes, you're welcome to do it in another PR after this is merged or as part of this one.